api/tests.py

```diff
@@ -149,6 +149,27 @@ def test_delete_trip(self):
 
         self.assertEqual(response.status_code, 404)
 
+    def test_post_session(self):
+        user = User.objects.create(name='Sam', email='swd1@gmail.com')
+
+        params = {'email': 'swd1@gmail.com'}
+        response = self.client.post('/sessions/', params)
+        data = response.data
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(data['email'], 'swd1@gmail.com')
+        self.assertEqual(data['name'], 'Sam')
+
+    def test_post_session_wrong_email(self):
```

Comment:
I like that you added the sad path test, definitely need to add this for the others too